converters/fromZigbee.js

```diff
@@ -2595,6 +2595,26 @@ const converters = {
             return result;
         },
     },
+    ts0219_siren: {
+        cluster: 'ssIasZone',
+        type: 'attributeReport',
+        convert: (model, msg, publish, options, meta) => {
+            const zoneStatus = msg.data.zoneStatus;
+            return {
+                alarm: zoneStatus === 17 ? true : false,
```

Comment:
Instead of `zoneStatus === 17 ? true : false`, does `(zoneStatus & 1) > 0` also work?

Reply:
have to check, the device is now on production in my house, so i can't check right now but i try to check ASAP.
I understand you prefer keep the same code as in another fz (i read a lot of time (zoneStatus & 1) > 0 in other fz) ?

Reply:
> I understand you prefer keep the same code as in another fz

Yes, this code is more robust

Reply:
> > I understand you prefer keep the same code as in another fz
> 
> Yes, this code is more robust

done, and it work as expected !